tests/pr_test.go

```diff
@@ -163,6 +163,57 @@ func TestRunUpgradeRoksPattern(t *testing.T) {
 	}
 }
 
+func TestValidateOverrideRoks(t *testing.T) {
+	t.Parallel()
+
+	options := setupOptionsRoksPattern(t, "slz-ovr-vld")
+	options.SkipTestTearDown = true
+	output, err := options.RunTestConsistency()
+
+	override_json := filepath.Join(roksPatternTerraformDir, "override.json")
+	config_json := options.LastTestTerraformOutputs["config"]
+	ignored_keys := []string{"resource_group"}
+
+	//Parse the json into a map
+	var override_json_map map[string]interface{}
+	err1 := json.Unmarshal([]byte(override_json), &override_json_map)
+	fmt.Println(err1)
```

Comment:
If the unmarshal has error, what should we do? is this a "FAIL and EXIT" condition? If so you can use the "require" package (see another comment below) to handle appropriately.

Reply:
used the require package